packages/core/src/desmosclient.ts

```diff
@@ -346,6 +347,14 @@ export class DesmosClient extends SigningCosmWasmClient {
         );
   }
 
+  /**
+   * Encode the given message objects into Amino messages.
+   * @param msgs: Messages to be encoded.
+   */
+  public encodeToAmino(msgs: readonly EncodeObject[]): AminoMsg[] {
```

Comment:
```suggestion
  protected encodeToAmino(msgs: readonly EncodeObject[]): AminoMsg[] {
```
I think that this can be `protected` instead of `public` since is not useful for who is using `DesmosClient`

Reply:
Actually, the idea is to allow developers to easily encode messages in the Amino format without having to register the various Amino types by themselves. This is particularly useful in situations where you want to encode those messages without having to sign the transaction (maybe because you later want to send such messages to external APIs). In this case, instead of creating a new `Registry` and then using that to perform the encoding, we can simply expose it publicly from the client and let the developers use this method instead.